Doxyfile.in

```diff
@@ -779,7 +779,6 @@ WARN_LOGFILE           =
 INPUT                  = @top_srcdir@/doc/coding_standard_c.md \
                          @top_srcdir@/doc/logging.md \
                          @top_srcdir@/doc/doxygen.dox \
-                         @top_srcdir@/README.md \
```

Comment:
You're still removing READM.md... ;-)

Reply:
because that's the README.md that causes the most warnings

Reply:
but it's also one of the most interesting one, right ? ;-)